e2e/nomostest/testing/env.go

```diff
@@ -16,6 +16,13 @@ package testing
 
 import "os"
 
+const (
+	// TestInfraContainerRegistry is the Config Sync test-infra registry hosted on GCR
+	TestInfraContainerRegistry = "gcr.io/kpt-config-sync-ci-artifacts"
```

Comment:
Since GCR will be deprecated soon, we can remove test with GCR hosts, but that would be a separate PR.